test/builds.test.js

```diff
@@ -200,6 +200,63 @@ describe('Builds', function () {
     });
   });
 
+  describe('.put', function () {
```

Comment:
@Swaagie there should be a test with a `@scoped/package` to make clear @epramer-godaddy's question. I know we typically need to encode for when it is a URL param so unsure about the decode here

Reply:
The encode happens in `_readParams` so this is me just inverting that again. Using that function made sense for the other components. I'll add the test though ;)